rita_exit/src/database/mod.rs

```diff
@@ -402,6 +437,33 @@ pub fn setup_clients(
     Ok(wg_clients)
 }
 
+pub fn get_client_interface(
+    c: &exit_db::models::Client,
+    new_wg_exit_clients: HashMap<WgKey, SystemTime>,
+    wg_exit_clients: HashMap<WgKey, SystemTime>,
+) -> Result<String, RitaExitError> {
+    match (
+        new_wg_exit_clients.get(&c.wg_pubkey.parse()?),
+        wg_exit_clients.get(&c.wg_pubkey.parse()?),
+    ) {
+        (Some(_), None) => Ok("wg_exit_new".into()),
+        (None, Some(_)) => Ok("wg_exit".into()),
+        (Some(new), Some(old)) => {
+            if new < old {
+                Ok("wg_exit_new".into())
+            } else {
+                Ok("wg_exit".into())
+            }
+        }
+        _ => {
+            error!("WG EXIT SETUP: Wg Exit not setup correctly. Client {}, does not have handshake with any wg exit interface", c.wg_pubkey);
```

Comment:
this could be true for any client that is down but registered. Is this trying to solve issues about where to set routes?

Reply:
This is the case where neither wg_exit nor wg_exit_new have a handshake, we just continue in this case